VoltDB.Data.Client/Types/ThirdParty.BigInteger.cs

```diff
@@ -2694,19 +2619,26 @@ public byte[] getBytes()
 
             byte[] result = new byte[numBytes];
 
-            //Console.WriteLine(result.Length);
-
             int pos = 0;
             uint tempVal, val = data[dataLength - 1];
 
             if ((tempVal = (val >> 24 & 0xFF)) != 0)
                 result[pos++] = (byte)tempVal;
+
             if ((tempVal = (val >> 16 & 0xFF)) != 0)
                 result[pos++] = (byte)tempVal;
+            else if (pos > 0)

```

Comment:
Yes, if in else the result[pos] should be 0. Byte[] when created, have all values in [] initialized to zero. So don't need to explicitly assign 0 for skipped positions.